packages/terra-heading/docs/heading-colors.md

```diff
@@ -1,3 +1,28 @@
 ## Heading Colors
 
-Using the `color` prop, authors can set the text color of any `<Heading />` component using a valid CSS color value.
+By passing in a className, authors can manually set the text color of any `<Heading />` component.
+
+```css
+:root {
+  --my-app-heading-attention-color: rgb(220, 20, 60);
+  --my-app-heading-neutral-color: rgb(46, 125, 50);
+  --my-app-heading-info-color: #304ffe;
+  --my-app-heading-default-color: #333;
+}
+
+.default {
+  color: var(--my-app-heading-default-color, #333);
+}
+
+.attention {
+  color: var(--my-app-heading-attention-color, rgb(220, 20, 60));
+}
+
+.neutral {
```

Comment:
```suggestion
.success {
```

lets update the class name to be `.success` to better match the green color, `.nuetral` is mentally misleading/confusing as average dev might read it, see it.

Reply:
Fixed in https://github.com/cerner/terra-core/pull/2211/commits/618ae6178edc4a734ef31a044c3bc8d570051b5c